README.md

```diff
@@ -33,7 +33,7 @@ To disable cloning and installing, use:
 
 Run
 
-```$ docker run --name ardrive-cli-bats --rm --init -td --mount type=tmpfs,destination=/home/node/tmp ardrive-bats-docker && sleep 20 ```
+```$ docker run --name ardrive-cli-bats --rm --init -tdi --mount type=tmpfs,destination=/home/node/tmp ardrive-bats-docker && sleep 20 ```
 
 This includes a sleep of 20 seconds to ensure Docker clones and builds. Please be aware that this docker will last till you shutdown your system OR you manually stop it (docker stop ardrive-bats)
```

Comment:
I'd remove the `sleep of 20 seconds to ensure Docker clones and builds`. It could take longer or even less so we are not actually ensuring that.

Reply:
> be aware that this docker will last till you shutdown your system

I thought it would persist if ran in detached mode /: 
Is there a way to do so?

Reply:
Without --rm